src/components/Editor.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 import CodeMirror from 'react-codemirror';
-import _ from 'lodash';
+import isEqual from 'lodash/isEqual';
```

Comment:
More imports means more load time. I think in the worker where it only uses a few that is fine but in proper that's a lot of imports that can be consolidated.